src/main/java/com/djrapitops/extension/GriefDefenderExtension.java

```diff
@@ -95,19 +125,49 @@ public long claimedArea(UUID playerUUID) {
                 .sum();
     }
 
+    @StringProvider(
+            text = "Claim Block",
+            description = "If Eco is being used, the Blocks player can buy will be showed,\n " +
+                    "or will show like [initial + Bonus + Accrued + Reaming]",
+            iconName = "map",
+            iconColor = Color.BLUE_GREY,
+            iconFamily = Family.REGULAR,
+            showInPlayerTable = true
+    )
+    public String claimBlock(UUID playerUUID) {
+        if (api.isEconomyModeEnabled()) {
+            return "Remaining Blocks: " + getPlayerData(playerUUID).getRemainingClaimBlocks();
+        }
+        return getPlayerData(playerUUID).getInitialClaimBlocks() + "+" +
```

Comment:
Avoid duplicate calls to `getPlayerData` by declaring the returned value as a variable - multiple calls can be very expensive if the other plugin uses database to get the player data.